tpl/tplimpl/template.go

```diff
@@ -224,6 +224,9 @@ func (t *templateExec) Execute(templ tpl.Template, wr io.Writer, data interface{
 		defer t.Metrics.MeasureSince(templ.Name(), time.Now())
 	}
 
+	if t.d.Cfg.GetBool("debug") {
+		t.d.Log.Debug().Printf("Execute template: %q", templ.Name())
```

Comment:
I'm not merging this debug statement (see my previous comment; in this case it's the cost of the `.GetBool`). I'm fine with the first change